README.md

```diff
@@ -37,9 +37,18 @@ This figure represents tikv-server software stack.
 
 When a node starts, the metadata of node, store and region are registered into PD. The status of each Region and Store is reported to PD regularly.
```

Comment:
When a node starts, the metadata of the Node, Store and Region are registered into PD. The status of each Region and Store is reported to PD regularly.